demos/SnakeGame/Input.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using SnakeGame.Models;
+
+namespace SnakeGame
+{
+    public static class Input
+    {
+        public static void GetInputDirection(ConsoleKey command, Position p)
+        {
```

Comment:
This method is not OO. Since this is a `Get` method, it shall return the result.